VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -149,13 +150,19 @@ subroutine transpmodel(invar, dim_invar, tp_netass, option1)
 
 !       * END OF DAY
 
-        call transpmodel_daily_step(tp_netass, option1)
 
+       if (option1 .eq. 5) then
+           if(nday .eq. 1) allocate( output_mat (21, c_testday ) )
+        call transpmodel_daily_step(tp_netass, option1, output_mat)
+       else
```

Comment:
I don't see a difference between option1 .eq. 5 and "else". It seems that the following actions are identical. Could you verify and simplify or document?